README.md

```diff
@@ -34,13 +34,14 @@ And then configure `package.json` to look like this:
 {
   "scripts": {
     "changelog": "uphold-scripts changelog $npm_package_version",
-    "lint": "uphold-scripts lint",
+    "lint": "uphold-scripts lint .",
+    "lint-staged": "lint-staged -q",
```

Comment:
Just a thought, what if we kept only the lint script and added the staged as an option on the script file. My thinking is to not pollute the scripts, and it doesn't seem that the file would become overly complicated.

Reply:
Yes we can do that. What would be the option name? Something like —files?

Reply:
Huh, I was thinking just call one or the other. But if we want to get creative, why not just an option `--staged` that would get the files from a git command to pass to eslint. It would avoid an unecessary dependency.

Reply:
I've talked with @Americas on slack about this, and because on how `pre-commit` works (we can only target scripts declared in the package.json), we wouldn't get much benefit and it would be harder to implement like we desired.

We decided to keep it like this for now, and evolve / improve this package later on, perhaps when the DX team kickstarts later in Q2.

Reply:
> ...when the DX team kickstarts later in Q2.

:eyes: :rocket: :tada: :clap: :heart:  @nunofgs @nunojsferreira @rplopes